### DISCUSSION TITLE
KML export

### DISCUSSION THREAD

**Discussion by ThoBur** [TomoyukiAota/photo-location-map#503] [Ideas]
Hi,

many thanks for the new version (1.7.0). Works perfect.

I have some feature requests which could be really useful. I have 3000 Images and would like to select and unselect som files and save the project at the end. When the project ist finished it would be great to share the project with friends or co-traveller. KML should support all needs. You can import KML in Google Maps and Google Eart or other Apps.

- Feature Requests
	- German Date Format DD.MM.YYYY HH:MM:SS
	- Save the current selection in a Project File for later revisit.
	- KML Export feature
		- with the following options:
			- first export parameter (combobox):
				- export all pins incl. collapsed pins
				- export only pins from the current zoom level. collapsed pins are one pin
			- second export parameter (checkbox):
				- export only selected pins
			- third export parameter (checkbox)
				- include thumbnail pic and gps data in the KML description
			- fourth export parameter (FileDialog->Textbox):
				- export path

Again many thanks for the tool.

regards,

ThoBur

**Comment by TomoyukiAota** [comment]
Thanks for your suggestions. I'll think about the suggestions as a future enhancement after working on other high-priority tasks.

**Reply by ThoBur** [reply]
Yes sure. All of this is nice to have stuff.
